models/role.ts

```diff
@@ -0,0 +1,6 @@
+export enum Role {
```

Comment:
Let's add role as a field in the user database too so we can store it and easily check it every time.

Reply:
https://github.com/uwblueprint/social-diversity-for-children/pull/18/commits/db29fdec0e99932472e54046dae54f53be52fcec